src/views/LoanDetails/LoanDetails.test.js

```diff
@@ -344,7 +360,7 @@ describe('LoanDetails', () => {
       userEvent.click(screen.queryAllByRole('button')[9]);
       expect(screen.getAllByText('ui-users.loans.markAsMissing')).toBeTruthy();
     });
-    it.skip('Fee Fine Else Condition More than 1 loanAccountActions', () => {
+    it('Fee Fine Else Condition More than 1 loanAccountActions', () => {
```

Comment:
Could you please improve the test title to better elucidate the aim of the unit test?